config/routes.rb

```diff
@@ -5,16 +5,35 @@
   # You can have the root of your site routed with "root"
```

Comment:
Your routes are set up SO WELL! nice work! The only other thing to be sure to be consistent with is whether or not you use the `/` at the beginning of the URL definition